Automattic/woocommerce-payments
# Remove wrong changelog file introduced in previous revert commit.

#### Changes proposed in this Pull Request









#### Testing instructions





* Remove the changelog file I mistakenly introduced in this revert [PR](https://github.com/Automattic/woocommerce-payments/pull/6918). The changelog of a revert PR should be empty to prevent it from showing in the changelog.txt and readme.txt for the next release

-------------------


**Post merge**
